docs/parameters.md

```diff
@@ -0,0 +1,180 @@
+      "payload": "Hello, Dorothy from Kansas"
+  }
+  ```
+
+### Setting default parameters on an action
+
+Actions can be invoked with multiple named parameters. Recall that the `hello` action from the previous example expects two parameters: the *name* of a person, and the *place* where they're from.
+
+Rather than pass all the parameters to an action every time, you can bind certain parameters. The following example binds the *place* parameter so that the action defaults to the place "Kansas":
+
+1. Update the action by using the `--param` option to bind parameter values, or by passing a file that contains the parameters to `--param-file`
+
+  To specify default parameters explicitly on the command-line, provide a key/value pair to the `param` flag:
+
+  ```
+  wsk action update hello --param place Kansas
+  ```
+
+  Passing parameters from a file requires the creation of a file containing the desired content in JSON format.
+  The filename must then be passed to the `-param-file` flag:
+
+  Example parameter file called parameters.json:
+  ```json
+  {
+      "place": "Kansas"
+  }
+  ```
+
+  ```
+  wsk action update hello --param-file parameters.json
+  ```
+
+2. Invoke the action, passing only the `name` parameter this time.
+
+  ```
+  wsk action invoke --result hello --param name Dorothy
+  ```
+  ```json
+  {
+      "payload": "Hello, Dorothy from Kansas"
+  }
+  ```
+
+  Notice that you did not need to specify the place parameter when you invoked the action. Bound parameters can still be overwritten by specifying the parameter value at invocation time.
+
+3. Invoke the action, passing both `name` and `place` values. The latter overwrites the value that is bound to the action.
+
+  Using the `--param` flag:
+
+  ```
+  wsk action invoke --result hello --param name Dorothy --param place "Washington, DC"
+  ```
+
+  Using the `--param-file` flag:
+
+  File parameters.json:
+  ```json
+  {
+    "name": "Dorothy",
+    "place": "Washington, DC"
+  }
+  ```
+
+  ```
+  wsk action invoke --result hello --param-file parameters.json
+  ```
+
+  ```json
+  {  
+      "payload": "Hello, Dorothy from Washington, DC"
+  }
+  ```
+
+### Setting default parameters on a package
+
+Parameters can be set at the package level, and these will serve as default parameters for actions unless:
+
+- the action itself has a default parameter
+- the action has a parameter supplied at invoke time, which will always be the "winner" where more than one parameter is available
+
+The following example sets a default parameter of `name` on the `MyApp` package and shows an action making use of it.
+
+1. Create a package with a parameter set
+
+ ```
+ wsk package update MyApp --param name World
+ ```
+
+2. Create an action in this package
+
+ ```
+    function main(params) {
+        return {payload: "Hello, " + params.name};
+    }
+ ```
+
+ ```
+ wsk action update MyApp/hello hello.js
+ ```
+
+3. Invoke the action, and observe the default package parameter in use
```

Comment:
Add colon --- Invoke the action, and observe the default package parameter in use: